### PR TITLE
Fix broken image redirect links on Chrome

### PR DESCRIPTION
**Pull request by kwhetstone**
Due to a recent change in Chrome, links to the main Jenkins project to get images were broken.  

> violates the following Content Security Policy directive: "img-src 'self'

This PR embeds the images as base64 since they're so small.  Also updates to the gae project to use the image internally; there is already an image repository.  Images now display across all browsers.

@reviewbybees

### CODE REVIEW

File: plugins-compat-tester-gae/src/main/webapp/index.html [hunk 1]
```diff
@@ -286,7 +286,7 @@
                     </table>
                     {{if buildLogPath != ""}}
                     <a name="${pluginName}_${pluginVersion}_${core}" />
-                    <a href="#${pluginName}_${pluginVersion}_${core}" onclick="javascript:displayLog('${pluginName} v${pluginVersion} against ${core}', '${buildLogPath}');"><img class="terminal" src="https://github.com/jenkinsci/jenkins/raw/master/war/src/main/webapp/images/24x24/terminal.png" /></a>
+                    <a href="#${pluginName}_${pluginVersion}_${core}" onclick="javascript:displayLog('${pluginName} v${pluginVersion} against ${core}', '${buildLogPath}');"><img class="terminal" src="img/pct/terminal.png" /></a>
```
Comment (recena) [inline plugins-compat-tester-gae/src/main/webapp/index.html]: @kwhetstone we are going to have the same problem with this PNG, right?

File: plugins-compat-tester/src/main/resources/resultToReport.xsl [hunk 2]
```diff
@@ -265,7 +265,20 @@ th.version {
 	    <xsl:attribute name="id"><xsl:value-of select="$id" /></xsl:attribute>
 	    <xsl:attribute name="alt"><xsl:value-of select="$title" /></xsl:attribute>
 	    <xsl:attribute name="title"><xsl:value-of select="$title" /><xsl:if test="not($error='_')"><xsl:value-of select="$error" /></xsl:if></xsl:attribute>
-	    <xsl:attribute name="src">https://github.com/jenkinsci/jenkins/raw/master/war/src/main/webapp/images/24x24/<xsl:value-of select="$img" /></xsl:attribute>
+        <xsl:choose>
```
Comment (recena) [inline plugins-compat-tester/src/main/resources/resultToReport.xsl]: Wrong indentation?

File: plugins-compat-tester/src/main/resources/resultToReport.xsl [hunk 3]
```diff
@@ -265,7 +265,20 @@ th.version {
...
+            </xsl:when>
+            <xsl:otherwise>
```
Comment (recena) [inline plugins-compat-tester/src/main/resources/resultToReport.xsl]: What is that case, `terminal.png`?

Comment (kwhetstone) [inline plugins-compat-tester/src/main/resources/resultToReport.xsl]: Yes.

File: plugins-compat-tester/src/main/resources/resultToReport.xsl (hunk 2 above)
Comment (kwhetstone) [inline plugins-compat-tester/src/main/resources/resultToReport.xsl]: The document is a mess in terms of tabs and spaces.  It appears there's a rogue tab in all the other lines throwing the doc out of alignment.  Fixing.

File: plugins-compat-tester-gae/src/main/webapp/index.html [hunk 4]
```diff
@@ -286,7 +286,7 @@
                     </table>
                     {{if buildLogPath != ""}}
                     <a name="${pluginName}_${pluginVersion}_${core}" />
-                    <a href="#${pluginName}_${pluginVersion}_${core}" onclick="javascript:displayLog('${pluginName} v${pluginVersion} against ${core}', '${buildLogPath}');"><img class="terminal" src="https://github.com/jenkinsci/jenkins/raw/master/war/src/main/webapp/images/24x24/terminal.png" /></a>
+                     <a href="#${pluginName}_${pluginVersion}_${core}" onclick="javascript:displayLog('${pluginName} v${pluginVersion} against ${core}', '${buildLogPath}');"><img class="terminal" src="https://github.com/jenkinsci/jenkins/raw/master/war/src/main/webapp/images/24x24/terminal.png" /></a>
```
Comment (daniel-beck) [inline plugins-compat-tester-gae/src/main/webapp/index.html]: Don't add files to a PR in which nothing is changed. https://github.com/jenkinsci/plugin-compat-tester/pull/21/files?w=1 doesn't have `index.html`

**Comment by daniel-beck** [comment]
Should be squashed after changes to index.html are gone.

File: plugins-compat-tester-gae/src/main/webapp/index.html (hunk 4 above)
Comment (kwhetstone) [inline plugins-compat-tester-gae/src/main/webapp/index.html]: I thought I had undone all the changes to this file.... ugh.  All these will definitely be squashed during merge.

**Comment by kwhetstone** [comment]
Knocked down to the one file.  I think that this is ready to merge.

**Comment by kwhetstone** [comment]
Are we ready to merge?

**Comment by daniel-beck** [comment]
> Should be squashed after changes to index.html are gone.

https://github.com/jenkinsci/plugin-compat-tester/commit/8def3c67dfeb5714fe40e85c96935b2305368ff2 :cry:

**Comment by daniel-beck** [comment]
@recena I asked that this be merged using squash (i.e. as one commit), but you merged as a merge commit. Now the back and forth is in the history files ultimately not modified (index.html, image files created and deleted again) by this PR.

**Comment by recena** [comment]
@daniel-beck I'm sorry.

**Pull request merged**